src/lib/src/core/viewer-service/viewer.service.ts

```diff
@@ -314,86 +326,122 @@ export class ViewerService implements OnInit {
   /**
    * Switches to DASHBOARD-mode, repositions pages and removes max-width on viewer
    */
-  toggleToDashboard(): void {
+  private toggleToDashboard(): void {
```

Comment:
toggleToDashboard can't be private because it is used by ViewerComponent unit test.

Reply:
I have fixed the test now.